web/src/environments/environment.prod.ts

```diff
@@ -19,6 +19,7 @@ import { PublicKHIExtension } from 'src/app/extensions/public/module';
 export const environment = {
   production: true,
   viewerMode: false,
+  apiBaseUrl: '',
```

Comment:
Just want to check if I understand it right. 
In dev profile, `this.baseUrl` = `http://localhost:8080` + " ".  
In prod, `this.baseUrl` = " " + `http://localhost:XXXX (host currently used)` ?

Reply:
I've added a comment on code to clarify this.
https://github.com/GoogleCloudPlatform/khi/pull/42/commits/bbdef50f272ebeb24f3873197ed3a52da01ca16b#diff-56a917cf4582773a0908e0f38e0e5aa50232d80507c5fdcf861e29b381fb3051R22

Your understanding is right, and this configuration varies to because of environment setup for development.